packages/api/src/command/medical/document/document-query.ts

```diff
@@ -49,6 +49,7 @@ export async function queryDocumentsAcrossHIEs({
   forceQuery = false,
   forceCommonwell = false,
   forceCarequality = false,
+  cqManagingOrgFilter,
```

Comment:
Should we indicate whether this is an ID or name? Like `cqManagingOrgName`

Reply:
Its a name - ill update